src/assets/fonts/selection.json

```diff
@@ -1,1582 +1 @@
-{
```

Comment:
What is that ? ^^

Reply:
It is the file for the icon library. I added two more that were needed for the keyboard